RecoVertex/BeamSpotProducer/plugins/BeamSpotAnalyzer.cc

```diff
@@ -82,12 +111,10 @@ void BeamSpotAnalyzer::beginLuminosityBlock(const edm::LuminosityBlock& lumiSeg,
 
 //--------------------------------------------------------
 void BeamSpotAnalyzer::endLuminosityBlock(const edm::LuminosityBlock& lumiSeg, const edm::EventSetup& iSetup) {
-  //LogDebug("BeamSpotAnalyzer") <<
-  std::cout << "for lumis " << beginLumiOfBSFit_ << " - " << endLumiOfBSFit_ << std::endl
-            << "number of selected tracks = " << theBeamFitter->getNTracks() << std::endl;
-  std::cout << "number of selected PVs = " << theBeamFitter->getNPVs() << std::endl;
-  //std::cout << "number of selected PVs per bx: " << std::endl;
-  //theBeamFitter->getNPVsperBX();
+  edm::LogPrint("BeamSpotAnalyzer") << "for lumis " << beginLumiOfBSFit_ << " - " << endLumiOfBSFit_ << std::endl
+                                    << "number of selected tracks = " << theBeamFitter->getNTracks() << std::endl;
+  edm::LogPrint("BeamSpotAnalyzer") << "number of selected PVs = " << theBeamFitter->getNPVs() << std::endl;
+  //edm::LogPrint("BeamSpotAnalyzer") << "number of selected PVs per bx: " << theBeamFitter->getNPVsperBX() std::endl;
```

Comment:
Is the `std::endl` needed when `edm::LogPrint` is used? I though the LogPrint takes care of the new line.
Also should the commented out line be removed?